pyquil/quil.py

```diff
@@ -181,7 +204,7 @@ def inst(self, *instructions: AbstractInstruction) -> "Program":
                     else:
                         params = []
```

Comment:
```suggestion
                         params: List[ParameterDesignator] = []
```

resolves the following mypy error:

```
pyquil/quil.py:205: error: Need type annotation for 'params' (hint: "params: List[<type>] = ...")
```

Reply:
Will appear in the 8th commit.